DEVELOPING.md

```diff
@@ -606,11 +606,14 @@ oc get pod %s -n %s -o go-template='{{len .spec.containers}}{{"\n"}}'
 
 ## Adding new handler
 
-To facilitate adding new handlers, the developers can leverage existing infrastructure to generate repetitive code.
-As an example, to generate handler with name Xyz, the command generate handler Xyz can be used(we need to run this command 
-in the folder "cmd/tnf/" and we run the command directly here).
+To facilitate adding new handlers, the "tnf" utility has been created to help developers to avoid generating repetitive code.
+As an example, to generate a new handler with named MyHandler, the command "tnf generate handler MyHandler" can be used.
```

Comment:
Add instructions to build or run the cmd/tnf/main.go. Use hyperlinks to reference files that are in the repo

Reply:
Done.